src/settings/target/manifest.rs

```diff
@@ -16,15 +15,27 @@ use serde::{Deserialize, Serialize};
 use crate::terminal::emoji;
 use crate::terminal::message;
 
+fn default_name() -> String {
```

Comment:
curious, why a fn and not a const?

Reply:
the diff is outdated, but the question is still valuable!